sdk/core/azure_core_amqp/src/value.rs

```diff
@@ -76,12 +76,17 @@ impl From<Vec<AmqpValue>> for AmqpList {
     }
 }
 
+/// AMQP Timestamp. Number of milliseconds since UNIX_EPOCH.
```

Comment:
Might want to clarify what the "UNIX epoch" instead of just mentioning its symbol (and not even in backticks.

Reply:
And as in some of the text below, rustdoc renders markdown so I'd reference symbols / expressions in backticks. Reads cleaner.